src/models/account/Address.ts

```diff
@@ -69,4 +74,12 @@ export class Address {
   public equals(otherAddress: Address) {
     return this.plain() == otherAddress.plain();
   }
+
+  /**
+   * Start listening new confirmed cache transactions
```

Comment:
Can we create a better description for this comment

Also this public confirmedTxObserver - is this something an developer is going to call? Does it need to be exposed.

Reply:
yes, this is the listener based off the address. Address.addObserver it used to be.